### DISCUSSION TITLE
'Creation of report schedule type 47700 is not allowed at this time' when creating GET_MERCHANT_LISTINGS_ALL_DATA schedule

### DISCUSSION THREAD

**Discussion by wolendranh** [saleweaver/python-amazon-sp-api#241] [Q&A] [answered]
Hi!

It is not possible to create report schedule for `GET_MERCHANT_LISTINGS_ALL_DATA` report as it results in error.

I used `sp_api.Reports.create_report_schedule` method.

API url used /reports/2020-09-04/schedules

Payload:
```
{'reportType': 'GET_MERCHANT_LISTINGS_ALL_DATA',
 'marketplaceIds': ['ATVPDKIKX0DER'],
 'period': 'P1D'}
```

I was able to create schedule for other report types, but not for `GET_MERCHANT_LISTINGS_ALL_DATA`

**Comment by saleweaver** [comment, marked as answer]
I don't think that you can schedule this report, the error message you get should say so as well. The report contains a lot of data that never or rarely changes, so scheduling this one instead of a report that contains just the data you need doesn't really make sense.

**Reply by wolendranh** [reply]
Hi @saleweaver ! Thx for response! What do you mean when saying 

> so scheduling this one instead of a report that contains just the data you need doesn't really make sense. ?

**Reply by saleweaver** [reply]
There are reports that contain just quantities, prices, ... that can be scheduled. Look what data you really need and schedule the report that contains only this data instead.
